### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by usingtechnology**
# Description
This is the first commit, just get a basic static placeholder site in place.
Basic BCGOV site with placeholder cards.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: _sass/header.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    img {
+        max-width: 100%;
+        margin-left: 0;
+        margin-right: 0;
+        margin-top: 0;
+        margin-bottom: 1.25rem;
+        padding-bottom: 0;
+        padding-left: 0;
+        padding-right: 0;
+        padding-top: 0;
```
Comment (loneil) [inline _sass/header.scss]: Can use the margin and padding shorthands here
https://alligator.io/css/padding-margin-shorthands/

File: _layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>{{ site.title }}</title>
+      <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```
Comment (loneil) [inline _layouts/default.html]: Indentation

File: _layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
```
Comment (loneil) [inline _layouts/default.html:4]: Might want the `<meta http-equiv="X-UA-Compatible" content="IE=edge"/>` header for BC Gov employees using their default IE11 (IE7 compatibility mode) settings. Unless that's provided somewhere by default on Github pages

**Review by loneil — CHANGES_REQUESTED** [review]
Minor questions

File: _sass/header.scss (hunk 1 above)
Comment (usingtechnology) [reply _sass/header.scss]: pushing commit up.  surprised that sass doesn't "fix" that...

File: _layouts/default.html (hunk 3 above)
Comment (usingtechnology) [reply _layouts/default.html:4]: good idea - in commit.

File: _layouts/default.html (hunk 2 above)
Comment (usingtechnology) [reply _layouts/default.html]: fixed in commit

**Review by jujaga — APPROVED** [review]
Looks good to me. We will need to add in Windows specific instructions so that we can get going on that platform in a future PR, but it looks good otherwise.

**Review by TimCsaky — APPROVED** [review]
works for me.
I used slightly different steps to install ruby and the required gems. 
looks like a good option.

**Pull request merged**